### DISCUSSION TITLE
What do you think about some gaming together?

### DISCUSSION THREAD

**Discussion by schmelto** [EddieHubCommunity/support#821] [Ideas] [answered]
Thought it would be fun if we play some games together like Among us or something else.

**Comment by Vyvy-vi** [comment]
discussion slightly related to this:  #785

**Comment by eddiejaoude** [comment, marked as answer]
Thanks @schmelto for the suggestion. I am keen to play http://battlesnake.io with the community - code and gaming 🎉 

@nhcarrigan is doing Among us gaming with the community

**Reply by Vyvy-vi** [reply]
I think we could build upon these and follow the path the above methods present...
Have some fun with games like AmongUs too but we also look into games that provide something related to our programming niche such as BattleSnake
